### DISCUSSION TITLE
使用taro3.6.4开发支付宝小程序主包跳转分包的时候报错__webpack_modules__[moduleId] is not a function

### DISCUSSION THREAD

**Discussion by Peter-Success** [NervJS/taro#13649] [Q & A]
打包出的微信小程序分包主包都没问题，但是支付宝主包跳转分包报错：__webpack_modules__[moduleId] is not a function，求大神帮忙

**Comment by LovesLolita** [comment]
我也有这个问题,引入dva.js出现的 但不能不用

**Comment by TheKonka** [comment]
https://github.com/NervJS/taro/issues/12562

dva 好像不更新了，能别用就别用

**Reply by LovesLolita** [reply]
有什么替代品吗
